models/stg_snapchat.yml

```diff
@@ -356,4 +368,177 @@ models:
+      - name: quartile_2
+        description: The number of times your video ad was viewed to 50% 
+      - name: quartile_3
+        description: The number of times your video ad was viewed to 75% 
+      - name: saves
+        description: Number of times components of the campaign were saved to Memories
+      - name: shares
+        description: Number of times components of the campaign were shared in a Chat or Story
+      - name: screen_time
+        description: The number of seconds spent watching your ad across all paid impressions. Screen time starts recording as soon as the media is fully rendered on the device and the autoplay video starts
+      - name: video_views
+        description: The number of times your video was watched at least 2 seconds or swiped up, whichever comes first
+      - name: view_completion
+        description: The number of times your video ad was viewed to completion (97% )
+      - name: view_time
+        description: The number of seconds spent watching your ad across all users reached.
+      - name: impressions
+        description: The number of impressions for an ad.
+      - name: spend
+        description: The amount of spend for an ad.
+      - name: swipes
+        description: The number of swipes for an ad. Swipes are when your ad was swiped up on or the CTA was tapped to view the attachment below.
+      - name: conversion_purchases_value
+        description: The total value of purchases attributed to an ad.
+      - name: conversion_purchases
+        description: The total number of purchases attributed to an ad.
+      - name: conversion_add_billing
+        description: Total number of attributed "ADD_BILLING" conversion events for an ad.
+      - name: conversion_save
+        description: Total number of attributed "SAVE" conversion events for an ad.
+      - name: conversion_add_cart
+        description: Total number of attributed "ADD_CART" conversion events for an ad.
+      - name: conversion_sign_ups 
+        description: Total number of attributed "SIGN_UP" conversion events for an ad.
+      - name: conversion_start_checkout
+        description: Total number of attributed "START_CHECKOUT" conversion events for an ad.
+      - name: conversion_searches
+        description: Total number of attributed "SEARCH" conversion events for an ad.
+      - name: conversion_view_content
+        description: Total number of attributed "VIEW_CONTENT" conversion events for an ad.
+      - name: _fivetran_synced
+        description: "{{ doc('_fivetran_synced') }}"
+      - name: conversion_app_opens
+        description: Total number of attributed "APP_OPEN" conversion events for an ad.
+      - name: conversion_level_completes
+        description: Total number of attributed "LEVEL_COMPLETE" conversion events for an ad.
+      - name: conversion_page_views
+        description: Total number of attributed "PAGE_VIEW" conversion events for an ad.
```

Comment:
I don't think these fields are included by default -- we should probably remove

Reply:
Ah I realized these were introduced for the ad, ad_squad, and campaign grain in the last release which is why they're here. But come to think of it, they should be removed entirely since they're not default fields